### PR TITLE
Minor fixes + add ability to run as script and option to set report title

### PR DESCRIPTION
**Pull request by andrewsiemer**
Closes:
- https://github.com/akavbathen/pytest_html_merger/issues/5
- https://github.com/akavbathen/pytest_html_merger/issues/6

### CODE REVIEW

**Comment by assadniang** [comment]
Hello,
How do we use the title argument to add a title to the merged report?
Or keep the title that is configured in the html report, it overtides that title.

**Comment by andrewsiemer** [comment]
@assadniang You can use "-t <new-title>" when using my fork

File: README.md [hunk 1]
```diff
@@ -8,13 +8,21 @@ This utility will merge all of your `pytest` HTML reports into a single HTML rep
 `pip install pytest-html-merger`
 
 ### running
-pytest_html_merger [-h] [--version] [-i INPUT] [-o OUTPUT]
+usage: main.py [-h] [--version] [-i INPUT] [-o OUTPUT] [-t TITLE]
```
Comment (progovoy) [inline README.md:11]: why changed
pytest_html_merger to main.py?!

**Comment by progovoy** [comment]
merged most of the features

**Comment by assadniang** [comment]
> @assadniang You can use "-t " when using my fork

I am getting this error message:

`(.venv) [an144a@DEVAUTOS02 tests]$ pytest_html_merger -t -i /opt/labs-scripts/python/pytest/nfv/suites/cisco/sdwan/pytest_output/reports/ -o /opt/labs-scripts/python/pytest/nfv/suites/cisco/sdwan/pytest_output/reports/test_report.html

usage: pytest_html_merger [-h] [--version] [-i INPUT] [-o OUTPUT]

pytest_html_merger: error: unrecognized arguments: -t
(.venv) [an144a@DEVAUTOS02 tests]$ `

**Comment by assadniang** [comment]
Upgraded to version 0.0.10 and -t worked

Thank you!

**Comment by assadniang** [comment]
Hello,
pytest_html_merger does not merge 2 tests if they using the same function

ie:
[cisco_sdwan_upgrade_C8500L8S4XBR60.html]
[cisco_sdwan_upgrade_NFX250U410BR20_.html]

These 2 test use the same funtion and file, but have different parameters (one is calling C8500L8S4XBR60 and the other NFX250U410BR20) as DUT

When I run pytest_html_merger, only one of these files shows up in the final report

**Pull request closed without merging**